src/manage_migrators.c

```diff
@@ -170,6 +167,8 @@ typedef struct
  * not just for migrators, so perhaps the SQL interface should keep
  * track of the transaction, and rollback before aborting. */
 
+// clang-format off
```

Comment:
INSERT_PERMISSION should be a function (insert_permission) instead of a macro, if that helps with formatting.

Reply:
Yes, it does. Added a commit to fix that.